rawpointer_test.go

```diff
@@ -232,3 +232,24 @@ func TestRawPointerTotalListSize(t *testing.T) {
 		}
 	}
 }
+
+func TestLandingPadNearPointer(t *testing.T) {
+	tests := []struct {
+		far     rawPointer
+		tag     rawPointer
+		landing rawPointer
+	}{
+		{0x08, 0x2000200000000, 0x2000200000000},
```

Comment:
Hey, trying to do as you suggest here but I have a problem understanding something, would love if you could clarify, in the function `rawFarPointer` you do:
`return farPointer | rawPointer(off&^7) | (rawPointer(segID) << 32)`. 
why the AND NOT in here? aren't you going against the spec here?

for example if I pass an offset of 160 to the function what I'll get is 20 in the offset section according to capnp spec and not the expected 160.

shouldn't be `return farPointer | rawPointer(off) << 3 | (rawPointer(segID) << 32)`?